docs/API-functions.md

```diff
@@ -515,6 +523,27 @@ The following group types are collected in an `SRT_GROUP_TYPE` enum:
 * `SRT_GTYPE_BACKUP`: backup type, idle links take over connection on disturbance
 * `SRT_GTYPE_BALANCING`: balancing type, share bandwidth usage between links
 
+The most important structure for the group members is `SRT_SOCKGROUPDATA`:
+
+```
+typedef struct SRT_SocketGroupData_
+{
+    SRTSOCKET id;
+    SRT_SOCKSTATUS status;
+    int result;
+    struct sockaddr_storage srcaddr;
+    struct sockaddr_storage peeraddr; // Don't want to expose sockaddr_any to public API
+} SRT_SOCKGROUPDATA;
+```
+
+where:
+
+* `id`: member socket ID
+* `status`: current connection status (see `srt_getsockstate`)
+* `result`: result of the operation (if this operation lately updated this structure)
```

Comment:
* `result`: result of the operation (if this operation has recently updated this structure)